.gitignore

```diff
@@ -2,4 +2,4 @@ node_modules
 .DS_Store
 dist
 dist-ssr
-*.local
\ No newline at end of file
+*.local
```

Comment:
Several files got changed just to ensure they end with an empty return (to make git happy about CRLF/LF).